deployment_scripts/appimage/build_appimage.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+python_cmd="python3.7"
+APPDIR="$HOME/monkey-appdir"
+INSTALL_DIR="$APPDIR/usr/src"
+
+GIT=$HOME/git
+
+REPO_MONKEY_HOME=$GIT/monkey
+REPO_MONKEY_SRC=$REPO_MONKEY_HOME/monkey
+
+ISLAND_PATH="$INSTALL_DIR/monkey_island"
+MONGO_PATH="$ISLAND_PATH/bin/mongodb"
+ISLAND_BINARIES_PATH="$ISLAND_PATH/cc/binaries"
```

Comment:
Do we want to remove all these dirs once appimage is built?

Reply:
Maybe, but I'm envisioning this being run in a clean VM or LXC and then being torn down. I'll add a README with this information.

Reply:
Oh, also, those are subdirectories in $APPDIR, which you generally want to remove between runs, but for development and debugging you don't.

Reply:
See the new README for info on cleaning up.